### ISSUE TITLE
ORCH-3: Support for Forwarded and X-Forwarded de-facto standards for proxy servers

### ISSUE THREAD

**Issue by pnicolucci** [OpenLiberty/open-liberty#5146]
Support for Forwarded and X-Forwarded de-facto standards for proxy servers. Aha feature: https://bigblue.aha.io/features/ORCH-3

**Comment by atosak** [comment]
Ship readiness checklist is being tracked here: https://github.com/OpenLiberty/open-liberty/issues/5451

**Comment by marikaj123** [comment]
Needs translation, waiting to provide the files for translation.

**Comment by gscottj** [comment]
Accessibility Approval Comment:
Based on the description provided, it appears that this feature does not have any end-user user interfaces that require accessibility compliance testing. If this assumption is not correct, then let me know that required accessibility verification testing of the UI will be performed before it is shipped and what are the results from those tests.

**Comment by sabolo** [comment]
I am doing a quick run through of features in advance of the Feature FAT approvals.  At present I am unable to find any links to a WAD, either from this Epic of in the AHA feature.  There is a link in AHA but it says its invalid.  Can you verify the link and ensure its accessible by all.  Many thanks

**Comment by sabolo** [comment]
I am looking into this feature for providing the Feature FAT approval.  At present I cannot find the Feature Test Summary issue that should be under this Epic.  Please can you add the issue as expected... If unsure on the process you can find it here:  https://github.ibm.com/was-liberty/WS-CD-Open/wiki/Feature-Review.  Many thanks

**Comment by atosak** [comment]
The team mentioned today that This feature is re-targeted to 1Q19.

**Comment by ebreijo** [comment]
*Serviceability Approval Comment* - Please answer the following questions for serviceability approval:

1.  WAD -- does the WAD identify the most likely problems customers will see and identify how the feature will enable them to diagnose and solve those problems without resorting to raising a PMR?  Have these issues been addressed in the implementation?
Yes, the WAD contains a Serviceability slide with a common problem and the solution. In addition, it contains the trace spec that needs to be enabled for problem determination. 

2.  Test and Demo -- As part of the new serviceability process we’re asking feature teams to test and analyze common problem paths for serviceability and demo those problem paths to someone not involved in the development of the feature (eg. L2, test team, or another development team).
a) What problem paths were tested and demonstrated?
b) Who did you demo to?
c) Do the people you demo’d to agree that the serviceability of the demonstrated problem scenarios is sufficient to avoid PMRs for any problems customers are likely to encounter, or that L2 should be able to quickly address those problems without need to engage L3?

3.  SVT -- SVT team is often the first team to try new features and often encounters problems setting up and using them.  Note that we’re not expecting SVT to do full serviceability testing -- just to sign-off on the serviceability of the problem paths they encountered.
a) Who conducted SVT tests for this feature? 
     Brian Hanczaryk
b) Do they agree that the serviceability of the problems they encountered is sufficient to avoid PMRs, or that L2 should be able to quickly address those problems without need to engage L3?

4.  Queues -- Which L2 / L3 queues will handle PMRs for this feature?  Are they listed in the contact reference pages? (https://w3-03.ibm.com/systems/techlink/crf/viewRecord.do?category=CRF&uid=724&sessionTime=1514919054315&rsid=0&seqNo=2&displayId=724&searchStr=websphere+application+server&totalHits=30)?  Do the respective L2 / L3 teams know they are supporting it?
    Cases and PMRs for the X-Forwarded-*/Forwarded  headers should be sent to the SalesForce WAS: L3 Channel Framework queue and Call Center WL3CHF,103 queue.

**Comment by pnicolucci** [comment]
No new Open source libraries or changes to existing libraries as part of this epic.

**Comment by marikaj123** [comment]
Translation files returned and feature owner did verify, approving!

**Comment by pnicolucci** [comment]
All approvals have been granted I'm going to close this epic.
